File: @LV1/1.2/cold/cold.go
package main

import "fmt"

func main() {
	var n, ans, tmp int
	fmt.Scanf("%d", &n)

	for i := 0; i < n; i++ {
		fmt.Scanf("%d", &tmp)
		if tmp < 0 {
			ans++
		}
	}

	fmt.Println(ans)
}

File: LV1/1.2/cold/cold.cpp
#include <cstdio>

int main()
{
  int n, ans = 0, tmp;

  scanf("%d", &n);

  for (int i = 0; i < n; i++)
  {
    scanf("%d", &tmp);
    if (tmp < 0)
      ans++;
  }
  printf("%d\n", ans);
}